.pylintrc

```diff
@@ -9,6 +9,6 @@ generated-members=numpy.*, torch.*
 
 
 [pylint.messages_control]
-disable=missing-function-docstring, line-too-long, import-error,
+disable=missing-function-docstring, line-too-long, import-error, too-many-lines,
```

Comment:
I added `too-many-lines` because `data.py` is already almost hitting the 1000 lines limit, and with puree dataset logic it exceeds the limit.